pkg/gather/clusterconfig/sap_license_management_logs.go

```diff
@@ -0,0 +1,46 @@
+package clusterconfig
+
+import (
+	"k8s.io/client-go/kubernetes"
+)
+
+// GatherSAPLicenseManagementLogs collects logs from license management pods with the following substrings:
+//   - "can't initialize iptables table",
+//
+// The Kubernetes API https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/pod_expansion.go#L48
+// Response see https://docs.openshift.com/container-platform/4.6/rest_api/workloads_apis/pod-core-v1.html#apiv1namespacesnamespacepodsnamelog
+//
+// Location in archive: config/pod/sdi/logs/{pod-name}/errors.log
+func GatherSAPLicenseManagementLogs(g *Gatherer, c chan<- gatherResult) {
+	defer close(c)
+	messagesToSearch := []string{
+		"can't initialize iptables table",
+	}
+
+	gatherKubeClient, err := kubernetes.NewForConfig(g.gatherProtoKubeConfig)
+	if err != nil {
+		c <- gatherResult{nil, []error{err}}
+		return
+	}
+
+	coreClient := gatherKubeClient.CoreV1()
+
+	records, err := gatherLogsFromPodsInNamespace(
+		g.ctx,
+		coreClient,
+		"sdi",
```

Comment:
Are you sure the namespace will always have this name?

Reply:
The SAP installation instruction uses this one, so it should always be the same.

Reply:
Well OK and what about if there's no SAP at all and someone has some `sdi` namespace? Then we will be scanning some pod logs in that as well right? I think I don't like this fact.

Reply:
@Sergey1011010 Have you talked to Michal or someone else from the SAP team about this? Because when I asked about the namespace name, he told me that it was arbitrary and that there could easily be multiple installations of SDI on the same cluster, each in its own namespace, of course. As far as I can tell, the namespace name is arbitrary and can be set to whatever the administrator wants. Therefore I don't believe we should many any assumptions about it.

Reply:
Check the new version please

Reply:
btw, we could also store container name in the path of the resulting file, but it would break back compatibility